jenkins-slaves/golang/Dockerfile.rhel7

```diff
@@ -0,0 +1,16 @@
+FROM cd/jenkins-slave-base
+
+LABEL maintainer="Michael Sauter <michael.sauter@boehringer-ingelheim.com>"
+
+ENV GO_VERSION 1.12.6
+
+RUN cd /tmp && \
+    curl -LO https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go$GO_VERSION.linux-amd64.tar.gz && \
+    rm -f *.tar.gz && \
+    cd && \
```

Comment:
@michaelsauter What's the point of the `cd` here? Without a parameter, this would just leave you where you are.

Reply:
`cd` without params brings you to you home directory. But yea, this is pretty pointless here. I will update to `cd -`, which brings you back to where you where before the last `cd`. I think that's good because then the `RUN` command does not change where you are ...

Reply:
Actually from a maintenance point, I would need to figure out where you are (because that is somewhere in the slave-base). My tendency would be to be as explicit as possible here (putting in an absolute path).

Reply:
@michaelsauter got it, thanks!

Reply:
@rattermeyer I'd consider using [`WORKDIR`](https://docs.docker.com/engine/reference/builder/#workdir) good practice (and is explicit).

Reply:
`WORKDIR` is already set. I will use `cd -`, which I still think is best.